includes/class-wc-stripe-intent-controller.php

```diff
@@ -745,15 +745,6 @@ public function create_and_confirm_payment_intent( $payment_information ) {
 			$request['confirm'] = 'false';
 		}
 
-		// Add the preferred credit card brand when defined
-		if ( isset( $payment_information['preferred_card_brand'] ) ) {
```

Comment:
Sorry, I moved this piece of code from here. It should be in the base branch. Since it will be merged to it and it still needs a review, I will leave it here for now, ok?

Reply:
Sounds good! Thanks for clarifying!